Cabal/Cabal.cabal

```diff
@@ -315,6 +315,7 @@ library
     Distribution.Compat.Exception
     Distribution.Compat.Graph
     Distribution.Compat.Internal.TempFile
+    Distribution.Compat.Prelude
```

Comment:
I'm not that found of Haddock-hiding, myself ;) And it doesn't help much once a snippet of code starts getting cargo culted around. I guess we could just slap a prominent comment saying that the API is NOT stable and should not be relied upon.